src/pages/components/style.scss

```diff
@@ -39,6 +28,14 @@
     }
 }
 
+.row-password-input, .row-date-input {
+    background-color: $primary-dark;
+    padding: 10px;
+    //Space to put away from other components (optional)
```

Comment:
If a css is optional it should be a new class so the user can decide if want to add or not.